find-phantom-resource.ps1

```
@@ -354,91 +369,125 @@ if ($SubScan) {
-                RgStatus      = $rgStatus
-                Id            = $r.id
-            })
+
+        # az resource list for comparison - may miss phantoms due to casing mismatch.
+        $azListJson = az resource list -g $rgName -o json --only-show-errors 2>&1
+        $azIds = @{}
+        if ($LASTEXITCODE -eq 0) {
+            try {
+                $azParsed = $azListJson | ConvertFrom-Json
+                foreach ($r in $azParsed) { $azIds[$r.id.ToLower()] = $true }
+            } catch {}
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)